podpac/core/settings.py

```diff
@@ -64,6 +65,8 @@ class PodpacSettings(dict):
 
     Attributes
     ----------
+    DEFAULT_COORD_REF_SYS : str
```

Comment:
this will be `DEFAULT_CRS` in #232 

I would actually just revert the changes in this file, or overwrite them with the changes in #232